src/window.rs

```diff
@@ -33,6 +33,9 @@ use crate::config::{Decorations, StartupMode, WindowConfig};
 #[cfg(windows)]
 static WINDOW_ICON: &'static [u8] = include_bytes!("../extra/windows/alacritty.ico");
 
+#[cfg(not(any(target_os = "macos", target_os = "windows")))]
+static WINDOW_ICON: &'static [u8] = include_bytes!("../extra/linux/alacritty.ico");
```

Comment:
Why can't we use the Windows icon here? I don't see any reason why this should be duplicated.

Reply:
For me it was strange to point to `windows` resources from linux feature and since I can't move icon to it's own folder ( due to windows rc files requirements) I decided to go this way. If using windows folder is ok, I'll change it.

Reply:
It should be possible to use relative paths for the window.rs file, right?

I also think that we shouldn't point to windows files for the linux target, but we can just use the `extra/logo` folder and put the ico there.

Reply:
[No](https://docs.microsoft.com/en-us/windows/desktop/menurc/icon-resource), that's why i did it this way.

Reply:
Microsoft strikes again I guess. In that case I'd go for just using the `.ico` from the windows directory. Maybe adding a comment that says it's required to be in that directory because of the .rc would help explain this in the future though.

Reply:
Would it make any sense to store a single file, and copy them into the appropriate locations in a `build.rs`, script? Too heavy?

Reply:
Yeah, at that point we're just adding way too much complexity for anyone to intuitively understand what's going on I think.

This way we have only one point where a workaround is in place and it allows us to document it at that same place too. So I think this is the best solution we have available right now.